multiplier/src/public.rs

```diff
@@ -18,12 +17,11 @@ pub struct Multiplier {
 
 impl Multiplier {
     // TODO Break this apart here
-    pub fn new() -> Multiplier {
+    pub fn new() -> color_eyre::Result<Multiplier> {
```

Comment:
```suggestion
    pub fn new() -> Result<Multiplier> {
```

Reply:
Done

Reply:
All these errors were fixed in [bddbe38](https://github.com/vacp2p/zerokit/pull/114/commits/bddbe38dc238d19d77d2441080dacf6b86d3b2cd)